src/components/binding/binding-panel.js

```diff
@@ -213,13 +187,44 @@ genotet.BindingPanel.prototype.updateTracks = function() {
       this.container.find('#genes #track-' + index).remove();
     }
   }
+
+  this.signal('loadBindingList');
+
+  this.container.find('#genes .glyphicon-remove')
+    .css('display', this.data.tracks.length == 1 ? 'none' : '');
+};
+
+/**
+ * Updates the track list for panel.
+ */
+genotet.BindingPanel.prototype.updateTrackList = function() {
```

Comment:
Well, you might be taking my words too aggressively. Consider your first response. The first paragraph points out correctly the loadBindingList stuff and this is a good response. However, the second paragraph rushes to assume that I'm doing "combined/uncombined" loops. Why not just asking me to double check the location of "loadBindingList"? At the very least you are the developer and I'm just a code reader. It would be more common that I missed something.

My responsibility is to spend a little bit of time to ensure code quality from 3rd person viewpoint. If you can argubaly write neat code that is convincingly good structured, I'd be happy to buy your decisions. Otherwise I will just point out the places I found potentially improper or suspicious. I never guaranteed that what I pointed out are absolutely justifiable, and sometimes they are just questions to re-confirm your decisions. You will always have the chance to revisit whatever I pointed out. Ultimately I'm gonna leave it to your sole discretion.